source/help/messaging/formatting-text.rst

```diff
@@ -125,7 +125,9 @@ Renders as:
 Code Block
 ----------
 
-Create a code block by indenting each line by four spaces, or by placing ``````` on the line above and below your code.
+Create a fixed width code block by indenting each line by four spaces, or by placing ``````` on the line above and below your code.
+ - This is best described as: three backticks, shift-enter, type stuff, shift-enter, three backticks, enter
```

Comment:
@fermulator this may not be correct if the user has "Send messages on CTRL-ENTER" on.

![image](https://user-images.githubusercontent.com/14333569/28219524-8d89a528-6879-11e7-9026-1f76c8a629d2.png)

Rather than an explanation of keystrokes, which is inconsistent with the rest of our doccumentation, maybe the best way to go about this would be to change the example text inside the code block to be a bit more human readable? For example:

```
this is my code
```